### DISCUSSION TITLE
I want to add .html after each route of nuxt3, is there a good way to implement this?

### DISCUSSION THREAD

**Discussion by yufanzheng0723** [nuxt/nuxt#16710] [Questions] [answered]
I thought of a stupid way to change page/xxx.vue to page/xxx.html.vue, which seems to work, but it is not so elegant. Is there any elegant way to implement it?

**Comment by TheAlexLichter** [comment, marked as answer]
You could use the `pages:extend` hooks and manipulate all routes at once there.

**Reply by yufanzheng0723** [reply]
wow good idea

**Reply by yufanzheng0723** [reply]
thank you very much for your help

**Reply by TheAlexLichter** [reply]
No problem 😋
